packages/odyssey-react-mui/src/labs/DataFilters.tsx

```diff
@@ -305,11 +308,24 @@ const DataFilters = ({
                     <Subordinate component="div">
                       {!latestFilterValue ||
                       (Array.isArray(latestFilterValue) &&
-                        latestFilterValue.length === 0)
-                        ? `Any ${filter.label.toLowerCase()}`
-                        : Array.isArray(latestFilterValue)
-                        ? `${latestFilterValue.length} selected`
-                        : latestFilterValue}
+                        latestFilterValue.length === 0) ? (
```

Comment:
I'm not sure if this is a bug or not, so please tell me if I got this wrong.
`latestFilterValue` is set with the following code:
```
const latestFilterValue = filters.find(
              (f) => f.id === filter.id
            )?.value;
```
Unless I'm mistaken, the type for `latestFilterValue` is either a `string`(I'm assuming `value` is of type `string`) or `undefined`. 
I'm suspecting in this code, `latestFilterValue` is being treated as an `Array`? 
The `string` type has a `length` property on it's prototype, but it's when I see `Array.isArray(latestFilterValue)` above and below this point that I'm a bit confused?
Can you clarify this?

Reply:
`latestFilterValue` may be one of three things:
1. `undefined`, for a filter that doesn't have a value set (which usually translates to something like "Any status" or "Any age"
2. A string, for filters that accept a single value (such as a search query)
3. An array, for filters that accept multiple values (such as checkboxes or multi-selects)